activerecord/lib/active_record/enum.rb

```diff
@@ -75,6 +75,19 @@ module ActiveRecord
   #
   #   Conversation.where("status <> ?", Conversation.statuses[:archived])
   #
+  # You can use <tt>:prefix</tt> option then you need to define multiple enums
+  # with same values. If prefix is <tt>true</tt>, the methods are prefixed with
+  # the name of the enum.
+  #
+  #   class Invoice < ActiveRecord::Base
+  #     enum({verification: [:done, :fail]}, prefix: true)
```

Comment:
In #17511 @dgsan had examples with visibility. It's be better if we define multiple enums where prefix will be needed. Maybe:
```
enum visible_for: [:admin, :user, :everybody]
enum editable_by: [:admin, :user, :everybody]
```
or something like this.